src/IBM.Cloud.SDK.Core/Authentication/CloudPakForData/CloudPakForDataAuthenticator.cs

```diff
@@ -145,6 +221,23 @@ protected DetailedResponse<CloudPakForDataTokenResponse> RequestToken()
                     request.WithHeaders(Headers);
                 }
 
+                List<KeyValuePair<string, string>> content = new List<KeyValuePair<string, string>>();
+                KeyValuePair<string, string> username = new KeyValuePair<string, string>(KeyUsername, Username);
+                content.Add(username);
+
+                if (string.IsNullOrEmpty(Password))
+                {
+                    KeyValuePair<string, string> apikey = new KeyValuePair<string, string>(KeyApikey, Apikey);
+                    content.Add(apikey);
+                } else
```

Comment:
ah this else should be on a separate line
```suggestion
                } 
                else
```